app/models/prison.rb

```diff
@@ -34,19 +79,15 @@ def visiting_slot_days
   end
 
   def anomalous_dates
-    (fetch(:slot_anomalies) { Hash.new }).keys.to_set
+    (slot_anomalies || {}).keys.to_set
   end
 
   def days_lead_time
-    fetch(:lead_days, DEFAULT_LEAD_DAYS)
-  end
-
-  def booking_window
-    fetch(:booking_window, DEFAULT_BOOKING_WINDOW)
+    lead_days
   end
 
   def works_weekends?
-    fetch(:works_weekends, false)
```

Comment:
If we have allowed Virtus to set a default value we don't need the fallback here.